pom.xml

```diff
@@ -330,6 +346,9 @@
 			</properties>
 			<build>
 				<plugins>
+					<!--
+					For the moment we do without
```

Comment:
TODO change the urls once we move this to eclipse org

Reply:
You mean all references to the current monorepo?
I left other TODOs concerning that.

Reply:
yes the new location will be `/xtext` as we had discussed before. but therefore sebastian first needs to do the archive step.